src/components/Layout/layout.js

```diff
@@ -1,7 +1,8 @@
 import React, { useState } from 'react'
 import Helmet from 'react-helmet'
 import { useStaticQuery, graphql } from 'gatsby'
-import { Header, Panel, Navigation } from '../../layouts'
+import { Header, Panel} from '../../layouts'
```

Comment:
Will need to fix this/run prettier before merging. I ran it in scope of #42 that picked up a lot of stray files, but it should fix this as well.